asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -302,11 +302,11 @@ docker run --rm -e ROOT_CONFLUENCE_URL=http://confluence-host \
    -e USERNAME=username \
    -e PASSWORD=1234 \
    -e SPACE_KEY=XYZ \
-   -e ANCESTOR_ID=012345 \
+   -e ANCESTOR_ID=012345 \bb
```

Comment:
What is the meaning of "bb"?

Reply:
That is an error, my bad. Will submit a fix shortly.